docs/how-to-migrate-from-smartlock.md

```diff
@@ -0,0 +1,791 @@
+|                                |     | leave password auth as an implicit    |
+|                                |     | "other option"; all OpenYOLO          |
+|                                |     | credentials must have a defined       |
+|                                |     | authentication method, which makes    |
+|                                |     | processing the credentials easier.    |
+|------------------------|---------------------------|-------------------------|
+| getPassword()          | getPassword()             |                         |
+|------------------------|---------------------------|-------------------------|
+| getName()              | getDisplayName()          |                         |
+|------------------------|---------------------------|-------------------------|
+| getProfilePictureUri() | getDisplayPicture()       |                         |
+|------------------------|---------------------------|-------------------------|
+| getIdTokens()          | getIdToken()              |                         |
+|------------------------|---------------------------|-------------------------|
+| getGeneratedPassword() | N/A  | As OpenYOLO credentials are specific to the  |
+|                        |      | existing account case, generated passwords   |
+|                        |      | are not applicable.                          |
+|------------------------|------|----------------------------------------------|
+| getFamilyName()        | N/A  | Not defined in the OpenYOLO specification,   |
+|                        |      | as this assumes too much culture-specific    |
+|                        |      | structure to names.                          |
+|------------------------|------|--------------------------------------------- |
+| getGivenName()         | N/A  | Not defined in the OpenYOLO specification,   |
+|                        |      | for the same reasons as for `getFamilyName`. |
+|-----|---------------------------|--------------------------------------------|
+| N/A | getAdditionalProperties() | OpenYOLO offers extensibility for Hints    |
+|     |                           | via a general map of strings to byte       |
+|     |                           | arrays.                                    |
+|-----|---------------------------|--------------------------------------------|
+
+
+## Saving Credentials
+
+When a credential is determined to be valid, it can be saved to the user's
+credential provider. In the Smart Lock for Passwords API, this was a two-step
+process similar to calling `retrieve()`:
+
+1. The credential is constructed and sent to Smart Lock for Passwords using the
+   `save()` method, with a callback provided. This callback is then invoked,
+   indicating whether the save succeeded, user consent is required to save
+   the credential (a `PendingIntent` is returned), or the save was rejected.
+
+2. If an Intent is returned, the developer must invoke this `PendingIntent` in
```

Comment:
Nit: PendingIntent is in backticks, but Intent is not

Reply:
Done, also searched for and fixed other occurrences of Intent and PendingIntent.